### PR TITLE
Features/search result skeleton 103

### PR DESCRIPTION
**Pull request by mballeza**

### CODE REVIEW

File: client/src/containers/searchresults/displaytable.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  return (
+    <Table selectable={false}>
+      <TableBody displayRowCheckbox={false} stripedRows>
+        {
+          props.users.map((user) => {
+            const isStriped = (index) => {
+              if (index % 2) return '#8195b1'
+              return 'white'
+            }
+            const isStripedButton = (index) => {
```
Comment (iandusenbury) [inline client/src/containers/searchresults/displaytable.js]: I'm pretty sure we can just do this in css. There is a :even/:odd pseudo selector
https://stackoverflow.com/questions/5080699/using-css-even-and-odd-pseudo-classes-with-list-items

File: client/src/containers/searchresults/displaytable.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                    </div>
+                    <FlatButton hoverColor={isStripedButton(user.id+1)} style={{float: 'left'}}>View profile</FlatButton>
+                  </div>
+                  <div className='table-about'>
+                    <ul>
+                      <li>Works at: { user.org_name }</li>
+                    </ul>
+                  </div>
+                  <ul className='table-contact'>
+                    <li>LinkedIn: { user.linkedin_id }</li>
```
Comment (iandusenbury) [inline client/src/containers/searchresults/displaytable.js]: I know this is all dummy data but if we want to be able to plug our stuff in then all our variables should be camel case so org_name -> orgName, if you want it to be really ready for actual data then use the exact names in the design doc (again, not sure if necessary right now)

File: client/src/containers/searchresults/index.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      <div className='search-header'>
+        <h1 className='search-title'>Search Results</h1>
+        <p>Results for { dummySearchKey }</p>
+      </div>
+      <div className='search-box'>
+        <div className='search-box-list'>
+          <DisplayTable users={dummyList} />
+        </div>
+        <div className='back-to-top'>
+          <FlatButton href='#top' flat>{'Return to top of results'}</FlatButton>
```
Comment (iandusenbury) [inline client/src/containers/searchresults/index.js]: I'm getting an error in the console here, it wants flat='true'

File: client/public/index.html [hunk 4]
```diff
@@ -10,6 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <!--link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css"-->
```
Comment (iandusenbury) [inline client/public/index.html]: Are we using this? if not remove line

File: client/package.json [hunk 5]
```diff
@@ -10,12 +10,14 @@
...
     "ramda": "^0.25.0",
     "react": "^16.2.0",
+    "react-bootstrap": "^0.32.0",
     "react-dom": "^16.2.0",
     "react-google-maps": "^9.4.5",
     "react-redux": "^5.0.6",
     "react-router-dom": "^4.2.2",
     "react-router-redux": "^5.0.0-alpha.9",
     "react-scripts": "1.0.17",
+    "react-toolbox": "^2.0.0-beta.12",
```
Comment (iandusenbury) [inline client/package.json]: remove line and run 'yarn' to regenerate yarn.lock

File: client/package.json [hunk 6]
```diff
@@ -10,12 +10,14 @@
     "material-ui-icons": "^1.0.0-beta.17",
     "ramda": "^0.25.0",
     "react": "^16.2.0",
+    "react-bootstrap": "^0.32.0",
```
Comment (iandusenbury) [inline client/package.json]: Are we using react-bootstrap? if not remove line and run 'yarn' to regenerate yarn.lock

File: client/src/containers/searchresults/displaytable.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
+import React from 'react'
```
Comment (iandusenbury) [inline client/src/containers/searchresults/displaytable.js]: I put this at the top so you'd see it first. The more we can abstract logic from the return function the better. You have all the pieces there and the map function is exactly what we want to do but here are my suggestions for this file as you have it now:
```
import React, { Component } from 'react'
import { bindActionCreators } from 'redux'
import { connect } from 'react-redux'
import {
  Table,
  TableBody,
  TableRow,
  TableRowColumn,
  Avatar,
  FlatButton,
} from 'material-ui'

class DisplayTable extends Component {
  constructor(props) {
    super(props)

    this.renderTableRows = this.renderTableRows.bind(this)
  }

  renderTableRows() {
    const { users } = this.props

    const tableRows = users.map((user) => {
      const {
        id,
        first_name,
        last_name,
        org_name,
        linkedin_id,
        contact_url
      } = user

      const isStriped = (index) => {
        if (index % 2) return '#8195b1'
        return 'white'
      }
      const isStripedButton = (index) => {
        if (!(index % 2))
        return '#f8f8f8'
      }

      return (
        <TableRow style={{backgroundColor: isStriped(id)}} key={id}>
          <TableRowColumn className='table-cell'>
            <div className='table-name' style={{borderBottomColor: isStriped(id+1)}}>
              <h2>{first_name} {last_name}</h2>
            </div>
            <div className='table-icon' style={{borderRightColor: isStriped(id+1)}}>
              <div style={{paddingLeft: '12px'}}>
                <Avatar size={60} src={require('./portrait.png')} />
              </div>
              <FlatButton hoverColor={isStripedButton(id+1)} style={{float: 'left'}}>View profile</FlatButton>
            </div>
            <div className='table-about'>
              <ul>
                <li>Works at: {org_name}</li>
              </ul>
            </div>
            <ul className='table-contact'>
              <li>LinkedIn: {linkedin_id}</li>
              <li>Contact: {contact_url}</li>
            </ul>
          </TableRowColumn>
        </TableRow>
      )
    })

    return tableRows
  }

  render() {
    return (
      <Table selectable={false}>
        <TableBody displayRowCheckbox={false} stripedRows>
          {this.renderTableRows()}
        </TableBody>
      </Table>
    )
  }
}

export default connect(
)(DisplayTable)

```

File: client/src/containers/searchresults/style.css [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+  .back-to-top {
+    margin: 10px 0px 5px 0px;
+    font-size: 14px;
+  }
+
+  ul {
+    padding-left: 8px;
+    padding-top: 4px;
+  }
```
Comment (iandusenbury) [inline client/src/containers/searchresults/style.css:149]: space after

File: client/src/containers/searchresults/displaytable.js [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+            }
+
+            return (
+              <TableRow style={{backgroundColor: isStriped(user.id)}} key={user.id}>
+                <TableRowColumn className='table-cell'>
+                  <div className='table-name' style={{borderBottomColor: isStriped(user.id+1)}}>
+                    <h2>{ user.first_name } { user.last_name }</h2>
+                  </div>
+                  <div className='table-icon' style={{borderRightColor: isStriped(user.id+1)}}>
+                    <div style={{paddingLeft: '12px'}}>
```
Comment (iandusenbury) [inline client/src/containers/searchresults/displaytable.js]: no inline styles, give it a class name and style with css

**Review by iandusenbury — CHANGES_REQUESTED** [review]

**Comment by mballeza** [comment]
Ready for another review.

**Review by iandusenbury — APPROVED** [review]

**Review by Domicilius — APPROVED** [review]

**Pull request merged**